src/components/Header/HeaderMobileClient.tsx

```diff
@@ -102,13 +101,9 @@ export default function HeaderMobileClient({
                                 />
                             )}
                             {data.isSignedIn && (
-                                <Link
-                                    href="/"
-                                    className="block hover:underline"
-                                    onClick={handleSignOut}
-                                >
+                                <button className="block hover:underline" onClick={handleSignOut}>
                                     Sign Out
-                                </Link>
+                                </button>
```

Comment:
Is there a reason why we couldn't use the `Button` component?

Reply:
That's because we don't want the `Button` component styling on the mobile menu. I had to use a button rather than a link like it was before so that it could call signIn('keycloak').

Reply:
Ah, fair enough